script/deploy.sh

```diff
@@ -1,16 +1,16 @@
 
 # Deploy contract
-forge script DeployScript --via-ir --optimizer-runs=999999 --rpc-url $RPC_URL --broadcast --private-key $PRIVATE_KEY
+forge script DeployScript --rpc-url $RPC_URL --broadcast --private-key $PRIVATE_KEY --verify --etherscan-api-key $ETHERSCAN_API_KEY
```

Comment:
is this because we're setting optimizer runs in foundry.toml?

Reply:
yep, wanted to keep variables like that to one place.